packages/material-ui/src/utils/setRef.js

```diff
@@ -1,4 +1,3 @@
-// TODO v5: make it private
```

Comment:
@oliviertassinari did you decide not to make this change anymore?

Reply:
Maybe not, let's keep it private for now, they might still be an opportunity to make it private.

Reply:
I meant, did you decide to remove the TODO because you don't want to do it in v5 anymore?

Reply:
I thought they were valid use cases for using it, however, looking at the codebase, I'm not really sure. I also know that @eps1lon wish to remove this module (should be in useForkRef).